roles/shiftstack/molecule/default/converge.yml

```diff
@@ -22,7 +22,8 @@
     cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
     cifmw_openshift_kubeconfig: "{{ ansible_user_dir }}/.crc/machines/crc/kubeconfig"
     cifmw_shiftstack_run_playbook: cifmw-gate.yaml
-    cifmw_run_test_shiftstack_testconfig: cifmw-gate.yaml
+    cifmw_run_test_shiftstack_testconfig:
```

Comment:
suggestion: Would it be a nice idea to add a second testconfig here so the molecule test can test this feature?

Reply:
Thanks for the suggestion, it made me think about the need of adding an index when the same test config is used. I added an index in the `testconfig_result_dict`:
```
ok: [localhost] =>                                         
  msg: |-                                                                                                                                                                                                                                     
    {                                                                                                                  
        "0": {                                                                                                                                                                                                                                
            "cifmw-gate.yaml": "PASS"                                                                                  
        },                                                                                                                                                                                                                                    
        "1": {                                                                                                         
            "cifmw-gate.yaml": "PASS"                                                                                  
        }                                                                                                              
    }  
```